### DISCUSSION TITLE
Bosswerk MI600 / Deye SUN600

### DISCUSSION THREAD

**Discussion by hburkert** [StephanJoubert/home_assistant_solarman#81] [Ideas]
My Bosswerk MI600 works fine with "deye-string.yaml". But you must supply your loggers serial-no and not inverter-serial-no in const.py

**Comment by ptalke** [comment]
If you want to additionally have the Temperature Reading from the Inverter, you can add the following to the yaml file:
Since there seems to be an offset of 1000, the measurement is 10 degrees to high.
Source: https://www.photovoltaikforum.com/thread/169125-answer-everything-about-deye-inverters/?postID=2722038#post2722038
    - name: "AC Radiator Temp - substract 10 degrees"
      class: "temperature"                       
      state_class: "measurement"                 
      uom: "C"                                   
      scale: 0.01                                
      rule: 1                                    
      registers: [0x005A]                        
      icon: 'mdi:thermometer'

**Comment by hex218** [comment]
Thanks! How did you find your inverter ip? My router won't show a new device - even though I'll get updates in solarman?!

**Reply by hburkert** [reply]
"Nearby" connect to your Deye-Inverter using WiFi  accesspoint APnnnnnnn. After connection enter address 10.10.100.1 in your browser.
Quickset shows IP settings.
![image](https://user-images.githubusercontent.com/47448353/198076590-32c9877b-70bc-497b-8fd5-43cf43d9f11c.png)
or analyze your network traffic with wireshark and search for connections 47.254.74.158 / access1.solarmanpv.com

**Reply by hburkert** [reply]
sorry, mistake: inverter address is 10.10.100.254 not 10.10.100.1

**Reply by hex218** [reply]
Thank you so much! Got it running now. 
I didn't recognize it in my DHCP because it was named "PC-192-168-[...]" and I have a few devices like that (strangely).
